arm/Microsoft.Network/virtualHubs/deploy.bicep

```
@@ -0,0 +1,143 @@
+@description('Required. The virtual hub name.')
+param name string
+
+@description('Optional. Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Tags of the resource.')
+param tags object = {}
+
+@description('Required. Address-prefix for this VirtualHub.')
+param addressPrefix string
+
+@description('Optional. Flag to control transit for VirtualRouter hub.')
+param allowBranchToBranchTraffic bool = true
+
+@description('Optional. Resource ID of the Azure Firewall to link to')
+param azureFirewallId string = ''
+
+@description('Optional. Resource ID of the Express Route Gateway to link to')
+param expressRouteGatewayId string = ''
+
+@description('Optional. Resource ID of the Point-to-Site VPN Gateway to link to')
+param p2SVpnGatewayId string = ''
+
+@description('Optional. The preferred routing gateway types')
+@allowed([
+  'ExpressRoute'
+  'None'
+  'VpnGateway'
+  ''
+])
+param preferredRoutingGateway string = ''
+
+@description('Optional. VirtualHub route tables')
+param routeTableRoutes array = []
+
+@description('Optional. ID of the Security Partner Provider to link to')
+param securityPartnerProviderId string = ''
+
+@description('Optional. The Security Provider name.')
+param securityProviderName string = ''
+
+@description('Optional. The sku of this VirtualHub.')
+param sku string = 'Standard'
```

This input supports basic and standard. Cannot find it in any other doc than [azcli](https://docs.azure.cn/zh-cn/cli/ext/virtual-wan/network/vhub?view=azure-cli-latest#ext-virtual-wan-az-network-vhub-create) or [terraform azurerm registry](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/virtual_hub)



```suggestion
@allowed([
  'Basic'
  'Standard'
])
@description('Optional. The sku of this VirtualHub.')
param sku string = 'Standard'
```